Xamarin.Forms.Controls.Issues/Xamarin.Forms.Controls.Issues.Shared/Xamarin.Forms.Controls.Issues.Shared.projitems

```diff
@@ -1157,4 +1160,15 @@
       <Generator>MSBuild:UpdateDesignTimeXaml</Generator>
     </EmbeddedResource>
   </ItemGroup>
-</Project>
+  <ItemGroup>
+    <EmbeddedResource Include="$(MSBuildThisFileDirectory)_TemplateMarkup.xaml">
+      <SubType>Designer</SubType>
+      <Generator>MSBuild:Compile</Generator>
+    </EmbeddedResource>
+  </ItemGroup>
+  <ItemGroup>
+    <Compile Update="C:\g\Xamarin.Forms\Xamarin.Forms.Controls.Issues\Xamarin.Forms.Controls.Issues.Shared\_TemplateMarkup.xaml.cs">
```

Comment:
ain't gonna fly (absolute path)

Reply:
also, why does this require an `Update` ?

Reply:
Not sure why this got added in the project file. Is the team still on VS2017? I recently switched to 2019.

Reply:
Seems like I can't set the custom tool to `MSBuild:UpdateDesignTimeXaml` and VS is adding an `Update` with the absolute path by default for any new XAML file added to the shared project.